jormungandr/src/settings/command_arguments.rs

```diff
@@ -45,6 +46,14 @@ pub struct StartArguments {
     pub explorer_enabled: bool,
 }
 
+#[derive(StructOpt, Debug)]
+pub struct RestArguments {
+    /// REST API listening address.
+    /// If not configured anywhere, defaults to REST API being disabled
+    #[structopt(long = "rest-listen-address")]
+    pub listen: Option<SocketAddr>,
```

Comment:
Nitpick: could be `--rest-listen` for brevity and commonality with config; also we have `*_address` settings that tend to be multiaddr and this is not (neither it should be).

Reply:
This was name proposed by @NicolasDP in the issue, but I agree with you, I'll fix it.